packages/assets-controllers/CHANGELOG.md

```diff
@@ -5,19 +5,51 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-- Update TokenRatesController to request token rates from the Price API in batches of 100 ([#3650](https://github.com/MetaMask/core/pull/3650))
+- **BREAKING:** `AccountTrackerController` constructor params object requires `getCurrentChainId` and `getNetworkClientById` hooks ([#3586](https://github.com/MetaMask/core/pull/3586))
+  - These are needed for the new "polling by `networkClientId`" feature
+- **BREAKING:** `AccountTrackerController` has a new required state property, `accountByChainId`([#3586](https://github.com/MetaMask/core/pull/3586))
+  - This is needed to track balances accross chains. It was introduced for the "polling by `networkClientId`" feature, but is useful on its own as well.
+- **BREAKING**: `AccountTrackerController` adds a mutex to `refresh` making it only possible for one call to be executed at time ([#3586](https://github.com/MetaMask/core/pull/3586))
+- **BREAKING**: `TokensController.watchAsset` now performs on-chain validation of the asset's symbol and decimals, if they're defined in the contract ([#1745](https://github.com/MetaMask/core/pull/1745))
+  - Additionally, if the symbol and decimals are defined in the contract, they are no longer required to be passed to `watchAsset`
+- **BREAKING:** Update controllers that rely on provider to listen to `NetworkController:networkDidChange` instead of `NetworkController:stateChange` ([#3610](https://github.com/MetaMask/core/pull/3610))
+  - The `networkDidChange` event is safer in cases where the provider is used because the provider is guaranteed to have been updated by the time that event is emitted. The same is not true of the `stateChange` event.
+  - The following controllers now accept a `onNetworkDidChange` constructor option instead of a `onNetworkStateChange` option:
+    - `TokensController`
+    - `AssetsContractController`
+- Update `@metamask/polling-controller` to v3 ([#3636](https://github.com/MetaMask/core/pull/3636))
+  - This update adds two new methods to each polling controller: `_startPollingByNetworkClientId` and `_stopPollingByPollingTokenSetId`. These methods are intended for internal use, and should not be called directly.
+  - The affected controllers are:
+    - `AccountTrackerController`
+    - `CurrencyRateController`
+    - `NftDetectionController`
+    - `TokenDetectionController`
+    - `TokenListController`
+    - `TokenRatesController`
+- Update `TokenListController` to fetch prefiltered set of tokens from the API, reducing response data and removing the need for filtering logic ([#2054](https://github.com/MetaMask/core/pull/2054))
+- Update `TokenRatesController` to request token rates from the Price API in batches of 100 ([#3650](https://github.com/MetaMask/core/pull/3650))
+- Add dependencies `cockatiel` and `lodash` ([#3586](https://github.com/MetaMask/core/pull/3586), [#3655](https://github.com/MetaMask/core/pull/3655))
```

Comment:
It looks like in addition to bumping `@metamask/polling-controller`, we also bumped `@metamask/controller-utils` to ^7.0.0. Should we include that too?

Reply:
Added here: https://github.com/MetaMask/core/pull/3667/commits/79aaee4709aacb815474fc8497f731c0e24ed0d0